app/models/user.rb

```diff
@@ -1,25 +1,26 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  before_create :create_role
-  has_many :posts
-  has_many :comments
-  has_many :users_roles
-  has_many :roles, through: :users_roles
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :nullify
```

Comment:
@sudakov-eugene you should do something with `PostsHelper#full_name` then.

Reply:
Ok! Thanks for let me know there are legacy code also! :+1:

Reply:
@sudakov-eugene You're welcome, thanks!
I wonder why you are surprised that there is a legacy code in the legacy code :stuck_out_tongue_winking_eye:

Reply:
Uhh, actually, there are nothing wrong and it's fine!